ros2_example_bazel_installed/ros2_example_apps/BUILD.bazel

```diff
@@ -200,6 +200,80 @@ ros_py_binary(
     ],
 )
 
+ros_py_binary(
+    name = "ap_custom_msg_rosbag",
+    srcs = ["ap_custom_msg_rosbag.py"],
+    data = [
+        # Provide both Python and C++ IDL to see if it can get picked up...
+        ":ros2_example_apps_msgs_py",
+        ":ros2_example_apps_msgs_cc",
+        # "@ros2//:rosidl_typesupport_cpp_cc",
+        # "@ros2//:rosidl_typesupport_cpp_py",
+        # "//ros2_example_common:ros2_example_common_msgs_copy_libs"
+
+        "//ros2_example_common:ros2_example_common_msgs_copy_cc_rosidl_typesupport_fastrtps_cpp",
+        "//ros2_example_common:ros2_example_common_msgs_copy_cc_rosidl_typesupport_introspection_cpp",
+        "//ros2_example_common:ros2_example_common_msgs_copy_cc_rosidl_typesupport_cpp",
+
+        "//ros2_example_common:ros2_example_common_msgs_copy_py_rosidl_typesupport_introspection_c",
+        "//ros2_example_common:ros2_example_common_msgs_copy_py_rosidl_typesupport_fastrtps_c",
+        "//ros2_example_common:ros2_example_common_msgs_copy_py_rosidl_typesupport_c",
+
+        "//ros2_example_apps:ros2_example_apps_msgs_copy_cc_rosidl_typesupport_fastrtps_cpp",
+        "//ros2_example_apps:ros2_example_apps_msgs_copy_cc_rosidl_typesupport_introspection_cpp",
+        "//ros2_example_apps:ros2_example_apps_msgs_copy_cc_rosidl_typesupport_cpp",
+
+        "//ros2_example_apps:ros2_example_apps_msgs_copy_py_rosidl_typesupport_introspection_c",
+        "//ros2_example_apps:ros2_example_apps_msgs_copy_py_rosidl_typesupport_fastrtps_c",
+        "//ros2_example_apps:ros2_example_apps_msgs_copy_py_rosidl_typesupport_c",
```

Comment:
Would be nice to add a bazel macro or some syntactic sugar around these deps, maybe for later

Reply:
https://github.com/RobotLocomotion/drake-ros/pull/317/commits/c7a84ff35f7ac8f64e6263cca6c3a2edf3e7a718